compiler/rustc_macros/src/diagnostics/diagnostic_builder.rs

```diff
@@ -44,9 +46,15 @@ pub(crate) struct DiagnosticDeriveVariantBuilder {
     /// has the actual diagnostic message.
     pub slug: SpannedOption<Path>,
 
+    /// Label is a the text embedded in the struct attribute and corresponds to the diagnostic
```

Comment:
Maybe `message` instead of `label` because `label` is the term we use for a type of subdiagnostic.